src/js/schemas/service-schema/ContainerSettings.js

```diff
@@ -24,6 +25,21 @@ const ContainerSettings = {
               return container.docker.image;
             }
             return null;
+          },
+          externalValidator: function ({containerSettings}, definition) {
+            const {image} = containerSettings;
+
+            if (image == null ||
+              ContainerValidatorUtil.isValidDockerImage(image)) {
+              return true;
+            }
+
+            definition.showError = 'Container Image, must not contain ' +
+              'whitespace, and should not contain any other characters ' +
```

Comment:
Not sure if the commas behind "Container Image" and "whitespace" and "underscores" are placed correctly.